packages/react/src/Button/Button.tsx

```diff
@@ -57,7 +57,7 @@ export const Button = forwardRef(
             </span>,
           ]
         default:
-          return [children, <Icon svg={icon} size="level-5" />]
+          return [children, <Icon key={2} svg={icon} size="level-5" />]
```

Comment:
Doesn’t have to be 2 in a different array.

```suggestion
          return [children, <Icon key={1} svg={icon} size="level-5" />]
```